packages/components/src/components/bal-list/bal-list-item-accordion-body/bal-list-item-accordion-body.tsx

```diff
@@ -33,7 +33,9 @@ export class ListItemAccordionBody {
   }
 
   componentDidRender() {
-    this.calcContentHeight()
+    setTimeout(() => {
```

Comment:
maybe to move it in componentDidLoad :)

Reply:
it's not working anymore with the new modification
<img width="1033" alt="Screenshot 2022-10-04 at 17 22 28" src="https://user-images.githubusercontent.com/17945926/193845292-78490bb5-8d44-428a-85db-45e6536da760.png">